### ISSUE TITLE
👋 Grall

### ISSUE THREAD

**Issue by ViolaineDo** [osm-fr/attributions#511]
### Ce signalement concerne…

- [ ] une application carto 

### Présence de l'attribution

❌ Pas d'attribution

### (Facultatif) Quel est le fond de carte ?

Autre/inconnu

### Prise de contact

✔️ Je l'ai contacté

### Informations supplémentaires

- Responsable contacté le : 18/10/2024
  - Responsable contacté par (courriel, Twitter…) : courril, fb
- Lié au signalement : #

**Comment by ViolaineDo** [comment]
> Il y a bien une attribution en bas du menu
> 
> ![Screenshot_20251113-100040.png](https://github.com/user-attachments/assets/e69a7c78-e22e-40a5-9ef0-19e75e6945fa)

Effectivement, peut être un correctif? De ma lecture des [recommandations d'attributions ](https://osmfoundation.org/wiki/Licence/Attribution_Guidelines#Interactive_maps)("Alternatively, the attribution may be placed adjacent to the map "), celà rentre dans les clous. Ce serait ok pour moi de fermer l'issue.

**Comment by Jean-BaptisteC** [comment]
Dans tous les cas, cela nécessite une interaction de la part de l'utilisateur donc ce n'est pas valide
